DigitalOffice/Services/FileService/Models/FileRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Rename directory to RestRequests, because in Models directory we store DTO models

Reply:
Renamed